main.h

```diff
@@ -42,6 +42,7 @@ typedef struct
     uint16_t proxy_port;
     uint8_t proxyenable;
     uint8_t logging_enabled : 1;
+    uint8_t close_to_tray;
```

Comment:
Use a bit from one of the 4 zero ones below to not break the utox save. Those are bitfields by the way.